### ISSUE TITLE
DeleteEvent not working on iOs 11.3

### ISSUE THREAD

**Issue by xale76** [EddyVerbruggen/Calendar-PhoneGap-Plugin#462]
Bug, deleteevent works on iOS 11.0 but returns “not able to add calendar event” on iOS 11.3.

**Comment by xale76** [comment]
nobody could help me?

**Comment by EddyVerbruggen** [comment]
We’re all eagerly waiting for your PR I guess.

**Comment by xale76** [comment]
What you mean please? I cannot understand

**Comment by thedewi** [comment]
@xale76 It means that if you would like to try fully diagnosing and patching the issue, it is likely to get addressed more quickly. I don't know when anyone else might find the resources to do it.

**Comment by xale76** [comment]
I already posted the problem. My source code didn't change from 2 years.

I create a Event by createEvent then I try to delete it by deleteEvent.
If I run my App on iOS 11 or earlier it works.
If I run my App on new iOs 11.3 it doesn't work: it's able to add event to calendar, but it issues an error "unable to add calendare event" when my app try to delete the event.

This is, I don't know what to explain more, please if I miss anything just ask!
Thank you anyway

**Comment by xale76** [comment]
I understand. But sorry I just filed a bug for this plugin. Should I solve it by myself? Or do you think that it's only me having this problem?
I really don't understand your answer. Not the same answer for other issues filed for this plugin. Sad
